internal/config/config.go

```diff
@@ -1043,6 +1045,14 @@ type AzureSEVSNP struct {
 	AMDRootKey Certificate `json:"amdRootKey" yaml:"amdRootKey"`
 }
 
+// modifyToMarshalNumericalVersionValues sets the WantLatest field to false for all versions in order to unmarshal the numerical versions instead of the string "latest".
+func (c *AzureSEVSNP) modifyToMarshalNumericalVersionValues() {
```

Comment:
I actually preferred the previous naming :sweat_smile: 
My comment was only regarding the method you added to set latest to false for the entire config